### ISSUE TITLE
Elixir recipes

### ISSUE THREAD

**Issue by windelbouwman** [joaohf/meta-erlang#27]
Hi,

Would this package at some point contain elixir recipes, or should that be another layer?

Regards,
Windel

**Comment by joaohf** [comment]
Hi Windel,

I don't see why not.

This layer has all the necessary pieces. Actually I have some plans to do but I haven't start anything yet.

Do you have any work about it?

A specific layer only to hold elixir potentially will be dependent from meta-erlang.

**Comment by windelbouwman** [comment]
I'll probably need it at some point in the coming months, so I will either check for it, or contribute it. Thanks!

**Comment by XDeschuyteneer** [comment]
My two cents: I have an application written in elixir (using this layer with thud release) and I created my own layer (to add machine, distro, recipes).

I just added this layer as a dependency in my own layer configuration.
If you take a look at meta-python, it's more or less like that too.
They integrate a lot of recipes, libraries, etc. but not everything.
If you want to add something of your own or just not included, the easiest way is to add it in your meta layer.
